src/app/modules/edit-user-nickname-modal/edit-user-nickname-modal.component.html

```diff
@@ -0,0 +1,14 @@
+<ion-list>
```

Comment:
Vi har faktisk en modal for å sette kallenavn allerede, se showSetNickDialog() i regobs-auth.service.
Denne brukes første gang man logger seg inn i regobs (som ny bruker).
Har du vurdert å gjenbruke denne?

Reply:
> Vi har faktisk en modal for å sette kallenavn allerede, se showSetNickDialog() i regobs-auth.service. Denne brukes første gang man logger seg inn i regobs (som ny bruker). Har du vurdert å gjenbruke denne?

https://github.com/NVE/regObs4/pull/702/files#r1995156679

Reply:
Jeg synes det hadde vært best å ha en felles løsning for dette. Supert om du du erstatter alerten med din løsning for å sette kallenavn!